lib/escape.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const escape = require('cross-spawn/lib/util/escape')
+
+/**
+ * Escape a shell argument for Unix shells
+ * @see https://qntm.org/cmd
+ * @param {string} arg the argument
+ * @returns {string} the escaped argument
+ */
+const escapeUnixShellArg = (arg) => `${arg.replace(/([^a-zA-Z0-9_])/g, '\\$1')}`
+
+/**
+ * Escape a shell argument for the Windows cmd.exe shell
+ * @see https://qntm.org/cmd
+ * @param {string} arg the argument
+ * @returns {string} the escaped argument
+ */
+const escapeWinCmdArg = (arg) => escape.argument(arg)
```

Comment:
Reuse the [cross-spawn](https://github.com/moxystudio/node-cross-spawn/blob/master/lib/util/escape.js#L13)'s escaper on Windows shells, instead of rewriting it.

Reply:
I was unable to get escaping work on Windows, so I left it out of this commit.